css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,72 @@
+.container {
+  width: 100%;
+  max-width: 1250px;
+  margin: auto;
+  background-color: white;
+}
+
+.row {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+@font-face {
+  font-family: 'DM Sans';
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
+}
```

Comment:
This can be removed, the link element for Google Fonts is already doing this for you.